### DISCUSSION TITLE
Contribution

### DISCUSSION THREAD

**Discussion by DarkLordGeo** [twbs/bootstrap#41031] [Q&A]
Hello people. so i am registered on codeTriage and i get daily issues on my gmail ,  most of them are dated 1 and 2 years ago, and beside that how can i contribute to bootstrap ?

**Comment by julien-deramond** [comment]
Hello @DarkLordGeo 
We have [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md) that are a good start for those willing to contribute.

**Reply by DarkLordGeo** [reply]
thank you, i will read that.
